ansible/tasks/logging/main.yml

```diff
@@ -0,0 +1,16 @@
+---
+- hosts: windows
+  gather_facts: no
+  tasks:
+    - name: Ping test
+      win_ping:
+- hosts: windows
+  vars:
+    ruby_version: "RubyInstaller-2.4.7-1/rubyinstaller-devkit-2.4.7-1-x86.exe"
```

Comment:
Do you want to match the versions of fluentd and ruby from linux host running in the cluster or would like to do that in a separate PR?

Reply:
I can address this in a separate PR and make the fluentd version here accepted from command line for now

Reply:
What is the reason for going with the 32-bit version when the 64-bit version is available?

Reply:
@aravindhp We are using 64 bit version. This path corresponds to 32 bit ruby installer (but we are installing dev kit instead), and I forgot to change this path.